websites/D/Dropout/presence.ts

```diff
@@ -0,0 +1,203 @@
+import { QueryParams, IFrameData } from "./interfaces";
+
+const presence = new Presence({
+		clientId: "1240164154682249227",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	logo = "https://i.ibb.co/hCK6Lj3/dropout-logo.png";
```

Comment:
```suggestion
	browsingTimestamp = Math.floor(Date.now() / 1000);
	
	const Enum Assets {
	Logo = "https://i.ibb.co/hCK6Lj3/dropout-logo.png",
	}
```

Reply:
Weird, I did this because my presence wouldn't compile during development because it couldn't find `Assets.Play` and `Assets.Pause` that I used in another function.

![image](https://github.com/PreMiD/Presences/assets/25342763/62e0e18b-d1a9-4b75-a7e6-816043350823)